Utilities/StaticAnalyzers/src/UnnecessaryMutableChecker.cpp

```diff
@@ -46,6 +46,15 @@ namespace clangcms {
       if (!Field->isMutable()) {
         continue;
       }
+
+      // == Skip non-private mutables
+      // Public mutable members are flagged by PublicMutableChecker
+      // Private ones can be modified by other classes, so it is bit hard to find if
```

Comment:
I'm confused with "**Private** ones can be modified by **other** classes"

Reply:
Reworded.